rotkehlchen/api/rest.py

```diff
@@ -1614,28 +1614,35 @@ def _get_history_debug(
             settings = self.rotkehlchen.get_settings(cursor)
             ignored_ids = self.rotkehlchen.data.db.get_ignored_action_ids(cursor, None)
         debug_info = {
-            'events': [entry.serialize() for entry in events],
+            'events': [entry.serialize_for_import() for entry in events],
             'settings': settings.serialize(),
             'ignored_events_ids': {k.serialize(): v for k, v in ignored_ids.items()},
         }
+        if filepath is not None:
+            with open(f'{filepath}/pnl_debug.json', mode='w') as f:
```

Comment:
Hmmm this is wrong. If a filepath is given then it should be the entire filepath. Here what is given is a directory and not a filepath. Can you change the logic so that what is provided is the entire filepath?